build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -62,9 +62,13 @@ public static BuildTargetResult Publish(BuildTargetContext c)
         [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
Can you add `BuildArchitecturesAttribute` rather than doing string matching? Or would the "current" platform still be x64 for the crossbuild, and only the target RID has the needed value?